sdk/keyvault/keyvault-keys/src/core/keyVaultClientContext.ts

```diff
@@ -40,8 +38,7 @@ export class KeyVaultClientContext extends coreHttp.ServiceClient {
     }
 
     if (!options.userAgent) {
-      const defaultUserAgent = coreHttp.getDefaultUserAgentValue();
-      options.userAgent = `${packageName}/${packageVersion} ${defaultUserAgent}`;
+      options.userAgent = USER_AGENT;
```

Comment:
Changing this isn't necessary since you're using your own userAgentPolicy in KeysClient.  This option doesn't even get used in that case.